docs/new-guide-how-to-provide-your-own-platform.md

```diff
@@ -0,0 +1,88 @@
+---
+id: new-guide-how-to-provide-your-own-platform
+title: How to Provide Your Own Platform
+---
+
+:::note What is this guide about?
+This guide will show you how to offer your own platform to application teams using meshStack. This enables you to provide platforms in an easy fashion including cost management, lifecycle management, self-service provisioning and enforce a compliant level of security and governance.
+:::
+
+
+On a high level, you need the following and your platform integration is done:
+
+1. Create a platform with its own name and icon.
+2. Create a Building Block that does the automation in the background using Terraform
+3. Create a Landing Zone that does the roll out of the Building Block.
+
+## Prerequisites
+
+- You have access to the Platform Builder in meshStack.
+- You have the necessary platform credentials and configuration details.
+- Terraform knowledge is helpful, as meshStack relies on Terraform for custom platform integrations.
+- There is some kind of platform/tenancy organization model in the cloud platform, just like e.g. Azure has an AAD (platform) and Subscriptions (tenant).
+- The above can be created & automated with Terraform.
+- (optional) Users and groups can be assigned to the tenant via Terraform (this allows you to do permission management as well)
+
+## Step-by-Step Guide
+
+### 1. Create a platform
+
+Follow [Create a Platform](new-guide-how-to-manage-a-platform.md) to create a new platform in meshStack.
+
+Make sure to:
+
+- Create a new platform type that matches your platform.
+- Make sure you customize the platform with a fitting name and and icon.
+
+✅ You now created a new platform with its own type.
+
+### 2. Create a Building Block
+
+Follow [Create a Building Block](new-guide-how-to-manage-building-blocks.md) to create a new Building Block that will do the automation for your platform.
+
+Make sure to configure the following details:
```

Comment:
This is a lot of info! I would rather link to a general building block guide and only highlight some aspects specific to what is interesting for custom platforms.

Reply:
On the other hand, it's there already so we can also keep using it.

Reply:
I took the existing documentation provided by the CR and shortened it quite a bit as it contained several pictures and Clcik next stuff. I think this is a very tricky thing to do if you are not already quite familiar with meshStack and Building Blocks.